.github/workflows/rake.yml

```diff
@@ -5,6 +5,7 @@ name: rake
 on:
   push:
     branches: [ master, main ]
+    tags: [ v* ]
```

Comment:
As I know tags are used to release gems. We test gems before releasing them. Running tests same time as releasing makes no sense unless releasing action waits until tests are completed successfully. Does release depend on test results? If not then please remove the `tags` trigger.

Reply:
> As I know tags are used to release gems. 

Yep, now we are going to a different approach:

- `rake` workflow will be triggered on push branches and tags too
- if tests passed rake will trigger `repository_dispatch` with `do-release` that will trigger actual release
- `release` workflow will not listen to `push tags` event anymore (this PR will be updated to ignore `push tags` for `release` workflow)

> Does release depend on test results?

Yes it will be

> Running tests same time as releasing makes no sense unless releasing action waits until tests are completed successfully

That is a good point, and this is how it suppose to work. Thanks for catching the problem

Reply:
Updated